components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/exception/ActionMgtException.java

```diff
@@ -31,16 +31,9 @@ public ActionMgtException(String message) {
         super(message);
     }
 
-    public ActionMgtException(String message, String errorCode) {
-
-        super(message);
-        this.errorCode = errorCode;
-    }
-
-    public ActionMgtException(String message, String errorCode, Throwable cause) {
+    public ActionMgtException(String message, Throwable cause) {
```

Comment:
Why was this needed ?

Reply:
To facilitate the requirement in ActionMgtServerException(String message, Throwable cause)
https://github.com/wso2/carbon-identity-framework/pull/6119/files#diff-2ca8badd17c1432b432842ffdf47a90fcc5e04d4dba336dc1539d6548af7f218R31:~:text=public%20ActionMgtServerException(String%20message%2C%20Throwable%20cause)%20%7B

Reply:
Is that needed and used ?

Reply:
Currently we are utlizing this to build exceptions in the ActionManagementDAO. We can remove this if we introduce new Exception type for the DAO. Will track that separately